dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store_vector_index.cpp

```diff
@@ -958,6 +896,111 @@ try
 }
 CATCH
 
+TEST_F(DeltaMergeStoreVectorTest, DDLAddMultipleVectorIndex)
+try
+{
+    {
+        auto indexes = std::make_shared<LocalIndexInfos>();
+        store = reload(indexes);
+        ASSERT_EQ(store->getLocalIndexInfosSnapshot(), nullptr);
+    }
+
+    const size_t num_rows_write = 128;
+
+    // write [0, 128) to store
+    write(0, num_rows_write);
+    // trigger mergeDelta for all segments
+    triggerMergeDelta();
+
+    // write [128, 256) to store
+    write(num_rows_write, num_rows_write * 2);
+
+    // trigger FlushCache for all segments
```

Comment:
```suggestion
    // trigger FlushCache for all segments
    // so that some data are stored at DeltaVS's persisted_file_set
    // that will build vector index
```

Reply:
no, there is no index info yet.